### DISCUSSION TITLE
Ant Design Mini 小程序组件库来了

### DISCUSSION THREAD

**Discussion by simplyme0823** [ant-design/ant-design-mini#8] [Announcements]
支付宝小程序官方 UI 组件库正式加入 Ant Design 大家庭，所有的组件都经过设计师的升级改造，既符合 Ant Design 的设计规范又满足大部分的开发需求。现在访问我们的[官网](https://mini.ant.design/)或 [Github](https://github.com/ant-design/ant-design-mini) 可以开始体验。
## 🏔 全新的设计体验
[Ant Design](https://ant.design/index-cn) Mini 是由支付宝设计团队基于蚂蚁集团的众多业务实践，亿万用户的验证反馈，抽象构建出的移动端资产库。希望可以帮助设计师们脑海中的 idea 轻松转变为自然好用的设计，给用户更真实自然的体验。

### 新 logo 新寓意新篇章
相信大家已经很熟悉 Ant Design 的品牌，Ant 家族已经有了 [Ant Design](https://ant.design/index-cn)、[AntV](https://antv.vision/zh) 等众多优秀的产品，今天新成员Ant Design Mini 也将和大家见面。新成员 Ant Design Mini 的 logo 概念来源于小程序英文“Mini Program”中首字母“M”的变形，同时，我们沿用了 Ant Design 系列 logo 的基础几何图形，希望达到更高的品牌认知度。

<img src="https://gw.alipayobjects.com/mdn/rms_e06820/afts/img/A*r9SDSafw6zsAAAAAAAAAAAAAARQnAQ"  width="100%"/>

### 追求极致完美的组件设计
组件的设计源于蚂蚁众多移动端业务的实践，包含但不限于支付宝基础、网商银行、区块链等各类业务。从中我们抽象提炼了 30+ 个基础通用的组件，这些组件包含导航、引导提示、信息展示、信息输入、反馈五大类型。在组件的视觉设计上，我们沿用了 [Alipay Design](https://design.alipay.com/) 最新的视觉风格，延续“暖科技”的视觉基调。

<img src="https://gw.alipayobjects.com/mdn/rms_e06820/afts/img/A*TfhCTaOea88AAAAAAAAAAAAAARQnAQ"  width="100%"/>

### 更多基础组件
当前基础组件数目及种类不足以覆盖大多数的移动端场景。因此，在未来几个月，我们将专注于组件库的扩充。

### 支持 CSS 变量
都2022 年了，是时候使用 CSS 变量来定制 UI 主题了。目前版本中有，但只有一点点 CSS 变量。因此我们没有在文档中透出相关内容。后续开发过程中，我们将在组件中大量使用 CSS 变量，并完全释放出来。

### 更好的文档
这次，我们更新了文档的结构与内容，demo 也更为详细，希望用户能通过文档获取更多有用的信息。

### 不断提升组件使用体验
更好的文档还不够好。如果不高兴看文档呢？那就直接书写吧。我们将会通过 VS Code 插件提供 axml 文件中组件的书写提示、属性补全及校验，进一步提升开发效率。

### 无障碍与大字版
Ant Design 是一个有温度的品牌。因此在下半年，antd-mini 会在无障碍与大字版方面进行探索。

### 最后
antd-mini 致力于提升支付宝小程序开发效率与使用体验。期待正式版的到来。
此外，[antd-mobile](https://mobile.ant.design/) 正式发布了，其作为 h5 组件库与 antd-mini 采用相同的设计规范。

**Comment by Me-UserName** [comment]
👍👍👍👍👍👍

**Comment by afc163** [comment]
ant-mini => antd-mini

**Reply by simplyme0823** [reply]
更新了~

**Comment by ShadyZOZ** [comment]
666
